torch/csrc/jit/passes/onnx/fold_if_node.cpp

```diff
@@ -123,7 +126,7 @@ static bool constantFoldedConditionValue(Node* node) {
   ScalarTypeAnalysisNodeForONNX(compare_node);
```

Comment:
I wonder if this pass and file `fold_if_node.cpp` is still used? I think it may be covered completely with shape related folding updates by David. cc @jiafatom

Reply:
Currently `fold_if` pass is still in use. My WIP PR https://github.com/pytorch/pytorch/pull/62856 is going to deprecate this pass - I did some minor changes, then we can deprecate this `fold_if` pass.

Reply:
Update: Deprecate `fold_if` pass PR https://github.com/pytorch/pytorch/pull/65697 is merged into `onnx_ms_1`.